SagaFlow.UI/src/Main.svelte

```diff
@@ -1,11 +1,21 @@
-<svelte:options customElement="sf-main"/>
+<svelte:options customElement={{
+    tag: "sf-main",
+    shadow: "none",
```

Comment:
not using shadow dom? is this for styling? Could we use the `:host` pseudo selector instead?

Reply:
Yes this is for styling, i'm planning to expose styles via well named css variables, maybe something like --sf-command-input-padding, to change the padding of the input elements, this will make styling mush easier without the use of the :host pseudo selector.